message_app_client/src/components/ConversationDetail.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState, useEffect } from 'react';
+import Messages from './Message';
+
+const ConversationDetail = (props) => {
+    const { messages, username } = props;
+
+    const renderMessage = (message) => {
+        const {id, sender_name, content, created_at} = message;
+        return (
+            <Messages
+                id={id}
+                sender_name={sender_name}
+                content={content}
+                created_at={created_at}
+                username={username}
+            />
```

Comment:
I think you have to leave a unique key for each Message or react will raise a warning.